src/extension.ts

```diff
@@ -78,6 +66,31 @@ function useStylefmt(document: vscode.TextDocument, range: vscode.Range): Promis
 		}));
 }
 
+/**
+ * Resolve Stylefmt config
+ */
+function useStylefmt(document: vscode.TextDocument, range: vscode.Range): Promise<IResult> {
+	const settings: IStylefmtOptions = vscode.workspace.getConfiguration().get('stylefmt');
+	const configResolver = new ConfigResolver(vscode.workspace.rootPath);
+	const options = {
+		packageProp: 'stylelint',
+		configFiles: [
+			'.stylelintrc',
+			'stylelint.config.js'
+		],
+		editorSettings: settings.config
+	};
+
+	return configResolver.scan(document.uri.fsPath, options).then((resolved) => {
+		return stylefmtProcess(document, range, {
+			configBasedir: settings.configBasedir || vscode.workspace.rootPath,
+			config: resolved.json
+		});
+	}).catch(() => {
+		return stylefmtProcess(document, range);
```

Comment:
I think that we shouldn't  work without config. This is abnormal situation and we need to inform the user. You can add https://github.com/mrmlnc/vscode-csscomb/blob/master/src/extension.ts#L11-L27 and replace this line on `showOutput(err.toString());`?